server/models/User.js

```diff
@@ -1,17 +1,19 @@
 const mongoose = require('mongoose');
 const { Schema } = mongoose;
+const { ObjectId } = Schema;
 const bcrypt = require('bcrypt');
 const saltRounds = 12; // defaults to 10
 
 const UserSchema = new Schema({
-  email: {
+  name: {
     type: String,
     required: true,
     unique: true
   },
```

Comment:
I changed name to not be unique, as I thought we discussed we can have similar names between users, but email unique